app.js

```diff
@@ -57,9 +58,8 @@ try {
 } catch (e) {
 	if (require.main !== module) throw new Error("Dependencies unmet");
 
-	let command = 'npm install --production';
-	console.log('Installing dependencies: `' + command + '`...');
-	require('child_process').spawnSync('sh', ['-c', command], {stdio: 'inherit'});
+	console.log('Installing dependencies...');
+	child_process.spawnSync('npm', ['install', '--production'], {stdio: 'inherit'});
 }
```

Comment:
In my sockets rewrite, I already do this since the launch script is necessary to recompile Go modules if any changes to them have been made. `npm start` also uses `node pokemon-showdown` instead of `node app.js` for the same reason